src/tap-click/activator.ts

```diff
@@ -41,7 +40,7 @@ export class Activator implements ActivatorBase {
     }
   }
 
-  downAction(ev: UIEvent, activatableEle: HTMLElement, startCoord: PointerCoordinates) {
```

Comment:
Wow I am not sure why did the IDE let me get away with this. reverted, tests passing